plugins/woocommerce/client/legacy/js/admin/woocommerce_admin.js

```diff
@@ -361,6 +361,26 @@
 			$( this ).trigger( 'focus' );
 		} );
 
+		const setPostThumbnail = $( '#set-post-thumbnail' );
+
+		if ( setPostThumbnail ) {
+			setPostThumbnail
+				.after(
+					`<span class="woocommerce-help-tip" tabindex="-1" for="content" aria-label="${ woocommerce_admin_meta_boxes.i18n_product_image_tip }"></span>`
+				)
+				.parent()
+				.find( '.woocommerce-help-tip' )
```

Comment:
The whole `.parent().find()` was bugging me, so I dug a bit...

you can do this instead...

```js
$( 
    `<span class="woocommerce-help-tip" tabindex="-1" for="content" aria-label="${ woocommerce_admin_meta_boxes.i18n_product_image_tip }"></span>`
)
    .insertAfter( setPostThumbnail )
    .tipTip()
```